data/sources/02_rooms-extended.yaml

```diff
@@ -1,125 +1,125 @@
+  name: Skriptenverkauf Fachschaft Mathe Physik Informatik Chemie / MPIC
   usage:
     name:
-      de: "Geschäftsraum"
-      en: "Retail space"
-    din_277: "NF4.5"
-    din_277_desc: "Verkaufsräume"
+      de: Geschäftsraum
+      en: Retail space
+    din_277: NF4.5
+    din_277_desc: Verkaufsräume
   props:
     links:
       - text:
-          de: "Öffnungszeiten & Artikel"
-          en: "Opening hours & articles"
-        url: "https://skripten.mpi.fs.tum.de/"
+          de: Öffnungszeiten & Artikel
+          en: Opening hours & articles
+        url: https://skripten.mpi.fs.tum.de/
 
 # === Chemie ===
 "5401.01.101I":
-  name: "5401.01.101I (CIP-Raum)" # Original name is "Rechenzentrum" ...
+  name: CIP-Raum # Original name is "Rechenzentrum" ...
 "5406.01.650C":
-  name: "5406.01.650C (Büro Fachschaft Mathe Physik Informatik Chemie / MPIC)" # Original name is "Büro" ...
+  name: Büro Fachschaft Mathe Physik Informatik Chemie / MPIC # Original name is "Büro" ...
 
 # === Maschinenwesen / MW ===
 "5501.EG.101":
-  name: "5501.EG.101 (Fachschaft Maschinenbau / FSMB)"
+  name: Fachschaft Maschinenbau / FSMB
 "5510.EG.011M":
-  name: "5510.EG.011M (Skriptenverkauf Fachschaft Maschinenbau / FSMB)"
-  short_name: "FSMB Skriptenverkauf"
+  name: Skriptenverkauf Fachschaft Maschinenbau / FSMB
+  short_name: FSMB Skriptenverkauf
   usage:
     name:
-      de: "Geschäftsraum"
-      en: "Retail space"
-    din_277: "NF4.5"
-    din_277_desc: "Verkaufsräume"
+      de: Geschäftsraum
+      en: Retail space
+    din_277: NF4.5
+    din_277_desc: Verkaufsräume
   props:
     links:
       - text:
-          de: "Öffnungszeiten & Artikel"
-          en: "Opening hours & articles"
-        url: "https://fsmb.de/studierende/skriptenverkauf/"
+          de: Öffnungszeiten & Artikel
+          en: Opening hours & articles
+        url: https://fsmb.de/studierende/skriptenverkauf/
 
 "5501.02.150":
-  name: "5501.02.150 (IKOM-Büro)"
+  name: IKOM-Büro
   usage: # previously: Meeting room
     name:
-      de: "Büro"
-      en: "Office"
-    din_277: "NF2.1"
-    din_277_desc: "Büroräume"
+      de: Büro
+      en: Office
+    din_277: NF2.1
+    din_277_desc: Büroräume
 "5504.EG.450":
-  name: "5504.EG.450 (Rechnerhalle)"
+  name: Rechnerhalle
 "5508.02.801":  # ID is probably wrong in TUMonline, but we need to use it for compatibility
-  name: "5508.01.801 (Ernst-Schmidt-Hörsaal)"
+  name: Ernst-Schmidt-Hörsaal
   props:
     comment:
-      de: "Es wird auch 5508.02.801 statt 5508.01.801 verwendet, obwohl der Hörsaal im 1. Stock ist."
-      en: "Also 5508.02.801 instead of 5508.01.801 is being used, even though the lecture hall is in the 1st floor."
+      de: Es wird auch 5508.02.801 statt 5508.01.801 verwendet, obwohl der Hörsaal im 1. Stock ist.
+      en: Also 5508.02.801 instead of 5508.01.801 is being used, even though the lecture hall is in the 1st floor.
```

Comment:
There is unfortunately the problem, that this way we cannot patch the ID in the title anymore. If you would now search for "5508.01.801" (the old and I think more correct ID), you wouldn't find the room anymore. Not sure of a clean solution, but I think there needs to be a way to make a patch to the full / final room name

Reply:
I think this should be handled via #30 or #199 and not via this mechanism
Furthermore, I am not convinced that people primarily use the IDs for finding rooms, but instead defer to using the Human name (in this case `MW1801`) so I would argue, that breaking this is nothing that should be a showstopper

The ways we could patch this involves a regex or dirty measures, which likely are going to cause problems in the future, which I would like to avoid such crude hacks. (If you think we should, we can put them in, but this should be an intentional choice…)

Reply:
The best way to fix would be to fix it upstream of course. The TCA app uses the ID for search in some functions, but then it's the ID from TUMonline. I just tested it with a different floor, and my assumption is probably wrong, it could be found (at least in the way search works currently):

![image](https://user-images.githubusercontent.com/7429408/217390005-51b2a139-28a6-4175-8949-3c599cce9a7b.png)

It's still that I don't like having data that is kind of wrong. At least these errors should be documented (It is at the moment, but only human-readable in the comment). Because if we don't, we'll forget and if we just take over everything as is from TUMonline, we end up with the same data quality. For example as soon as we have floorplans for the MW upper floors, it might display the wrong floor map, because it's the wrong floor in TUMonline.

So I'd say we can leave out a feature to patch the final name, but for the future (like when it comes to the floor etc.) we will need to have patching systems.